server/src/routes/api/v1/activityReviewRouter.js

```diff
@@ -58,4 +53,21 @@ activityReviewRouter.delete("/:id", async (req, res) => {
     }
 })
 
+activityReviewRouter.patch("/:id", async (req,res) =>{
+    const reviewId = req.params.id
+    const editedReview = cleanUserInput(req.body)
+
+    try {
+        await Review.query().findById(reviewId).patch({ content : editedReview.content })
+        const review = await Review.query().findById(reviewId)
```

Comment:
we can make this into one line with `pathcAndFetchById` 
https://vincit.github.io/objection.js/api/query-builder/mutate-methods.html#patchandfetchbyid